api/app/src/Migrations/Version292.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version292 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE court_order ALTER court_order_uid TYPE BIGINT USING court_order_uid::bigint');
```

Comment:
I'm not sure why the type of this field changed. It should be a varchar I think?

Reply:
It was, but the type was updated on doc hint to bigint